src/modules/interfaces.ts

```diff
@@ -299,7 +299,7 @@ export class MapperBase<E> {
       this.db.entityId = this.entityId;
       this.db.dest = 'db';
       this.db.module = this.module;
-    } else if (!!cloudColumn) {
+    } else if (!!cloudColumn && !((cloudColumn as any) instanceof Error)) {
```

Comment:
This is the bugfix. `getCloudId` doesn't return `undefined` when not specified, it returns an `Error` object instead, which is truthy.

Reply:
how did you catch it??

Reply:
When converting this module I forgot to include the `module` property on the mapper and was getting errors that didn't make sense, so I read through the initialization logic *and* the functions it calls, noticing this error and fixing it but it didn't fix the bug I was seeing, and then I finally realized what I had done wrong during the conversion process.